modules/adliveplusBidAdapter.md

```diff
@@ -0,0 +1,29 @@
+# Overview
+
+Module Name: Adlive Plus Bidder Adapter
```

Comment:
@aymeric45 Since you're aliasing another adapter, you can remove this adapter markdown file